ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -2852,7 +2851,7 @@ private static void GenerateTableDefinitionPartContent(TableDefinitionPart table
                     var baseField = pt.SourceRange.Columns().FirstOrDefault(c => c.Cell(1).Value.ToInvariantString() == value.BaseField);
                     if (baseField != null)
                     {
-                        df.BaseField = baseField.ColumnNumber() - 1;
+                        df.BaseField = baseField.ColumnNumber() - pt.SourceRange.RangeAddress.FirstAddress.ColumnNumber;
```

Comment:
I pushed this commit to fix your base field issue. Please `git pull --rebase` the changes on your side.

Reply:
Thanks